src/azure-iot-edge-installer.sh

```diff
@@ -1,6 +1,20 @@
-#script 
+# script 
 echo "Running azure-iot-edge-installer.sh"
 
+# if helper scripts dont exist, fetch via wget 
+if [ -d "iot-edge-installer" ] 
+then
+    echo "Directory iot-edge-installer exists." 
+else
+    mkdir iot-edge-installer
+    cd iot-edge-installer
+    wget https://github.com/Azure/iot-edge-config/releases/latest/download/validate-tier1-os.sh
```

Comment:
>latest [](http://example.com/codeflow?start=59&length=6)

should the script pull the version of the components that map to its own version?